src/ralph/dashboards/management/commands/push_graphs_to_statsd.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import logging
+import string
+import textwrap
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from ralph.dashboards.models import Graph
+from ralph.lib.metrics import statsd
+
+logger = logging.getLogger(__name__)
+PREFIX = settings.STATSD_GRAPHS_PREFIX
+STATSD_PATH = '{}.{{}}.{{}}'.format(PREFIX)
+
+
+def normalize(s):
+    allowed = list(string.ascii_letters + string.digits) + ['_']
+    s = s.lower()
+    s = s.replace(' ', '_')
+    s = s.replace('-', '_')
+    return ''.join([c for c in s if c in allowed])
+
+
+class Command(BaseCommand):
+    """Push to statsd data generated by graphs."""
+    help = textwrap.dedent(__doc__).strip()
+
+    def handle(self, *args, **kwargs):
+        graphs = Graph.objects.filter(push_to_statsd=True)
+        for graph in graphs:
+            graph_data = graph.get_data()
+            graph_name = normalize(graph.name)
+            for row in zip(graph_data['labels'], graph_data['series']):
```

Comment:
consider `key, value` or `label, value` instead of row (i mean unpacking this tuple in loop)